src/page/personal-information.page.ts

```diff
@@ -0,0 +1,60 @@
+import { $, by, ElementFinder, ExpectedConditions, browser } from 'protractor';
+
+export class PersonalInformationPage {
+  private firstNameBox: ElementFinder;
+  private lastNameBox: ElementFinder;
+  private sexCheckBox: ElementFinder;
+  private yearsOfExperienceCheckBox: ElementFinder;
+  private professionCheckBox: ElementFinder;
+  private toolSeleniumCheckBox: ElementFinder;
+  private continentSelect: ElementFinder;
+  private commandsSelect: ElementFinder;
+  private submitBtn: ElementFinder;
+  private timeout: number = 5000;
+
+  constructor () {
+    this.firstNameBox = $('input[name="firstname"]');
+    this.lastNameBox = $('input[name="lastname"]');
+    this.submitBtn = $('button[name="submit"]');
+  }
+
+  public async finishForm() {
+    await this.submitBtn.click();
+  }
+
+  public async fillForm(info: Object): Promise<void> {
```

Comment:
Instead of receiving a generic Object here, I would prefer to create a DataHolder or Bean class that will have as attributes the info needed in this step. Then you can specify to receive that object here. This change is not required, it is up to you.

Reply:
I’m agree..try to use interfaces or destructuring instead

Reply:
Yes, I knew about it, I did it in that way because time ago I heard that JavaScript allows to treat an object like a dictionary, so I wanted to try it.

Reply:
yep, but you are not taking advantage of the Typescript's powerfulness...What is the typescript's value if you are not handling types?

Reply:
Think about java and using Object instead of classes

Reply:
Done